compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/PropertyReferenceLowering.kt

```diff
@@ -95,6 +99,33 @@ internal class PropertyReferenceLowering(val context: JvmBackendContext) : Class
             calculateOwner(expression.propertyContainer, this@PropertyReferenceLowering.context)
         }
 
+    private fun IrBuilderWithScope.computeSignatureString(expression: IrCallableReference): IrExpression {
+        return expression.getter?.let {
+            localPropertyIndices[it]?.let { irString("<v#$it>") }
+                ?: if (it.owner.origin == IrDeclarationOrigin.DEFAULT_PROPERTY_ACCESSOR) {
+                    // Default property accessor. Compute the signature now, so that we will not get into trouble
+                    // if the getter is transformed to a static method by inline classes lowering.
+                    irString(expression.signature)
+                } else {
+                    // Delay the computation of the signature until after inline classes lowering to make sure
+                    // we mangle the function names correctly for things like extension methods on inline classes.
+                    irCall(signatureStringIntrinsic).apply {
+                        putValueArgument(
```

Comment:
Minor, but could you please join this into a single line for easier reading :)

Reply:
Of course! Done. :)